### PR TITLE
SearchPanel: Added a close button at the bottom.

### PR DESCRIPTION
**Pull request by PawBud**
This PR adds the close button on the Search Panel as shown [here](https://gofile.io/d/P9NPhO).

@4ian, is the button UI ok? I am a bit skeptical about its looks. Moreover, the button stays the same for all the themes.

### CODE REVIEW

**Comment by 4ian** [comment]
Thanks for the PR :) Could we see if in material-ui icons there is an icon showing something about "closing a pan", "reducing", "minimizing"? I feel it's taking too much space and visual attention. An icon like the previous/next buttons would be cool I think :)

**Comment by PawBud** [comment]
Screenshot from 2020-08-05 19-49-03
Screenshot from 2020-08-05 19-48-38

**Comment by Bouh** [comment]
Add a simple close button in plain text please.
The close button like in AboutDialog component;

```
<FlatButton
            key="close"
            label={<Trans>Close</Trans>}
            primary={false}
            onClick={onClose}
          />
```

**Comment by 4ian** [comment]
> The close button like in AboutDialog component;

Well that what PawBud did first, but I was thinking this was taking too much space. See his video. What do you think?

**Comment by 4ian** [comment]
A my bad, the first button was a `RaisedButton`. I'm fine with a `FlatButton` then! :)

**Comment by PawBud** [comment]
Done @4ian @Bouh https://gofile.io/d/RjJLLr, Does it look ok?

**Comment by Bouh** [comment]
> Done @4ian @Bouh https://gofile.io/d/RjJLLr, Does it look ok?

Yes 👍 
This fit to my mockup well done !

**Comment by Bouh** [comment]
@PawBud can you run prettier ?
`yarn run format`

File: newIDE/app/src/EventsSheet/SearchPanel.js [hunk 1]
```diff
@@ -253,6 +255,12 @@ export default class SearchPanel extends PureComponent<Props, State> {
               >
                 <ChevronRight />
               </IconButton>
+              <FlatButton
+                key="close"
+                label={<Trans>Close</Trans>}
+                primary={false}
+                onClick={()=>{this.props.onCloseSearchPanel()}}
```
Comment (Bouh) [inline newIDE/app/src/EventsSheet/SearchPanel.js:262]: You can remove `this.props.`

But for that you need add the props `onCloseSearchPanel `in the destructuring drops at line 120 of this file.

Comment (Bouh) [reply newIDE/app/src/EventsSheet/SearchPanel.js:262]: Take a look on `onGoToPreviousSearchResult`, how it's define at line 120, and how it's used :)

**Comment by Bouh** [comment]
Can you also add the new props in the stories ?
It is easy, search all instance of the SearchPanel component in the **newIDE/app/src/stories** directory and add the props wil an empty arrow function.

**Comment by PawBud** [comment]
Everything has been done as u requested @Bouh, thanks a lot for explaining every step carefully and pointing the mistakes. Pls, let me know if there is anything left to be done.

**Pull request merged**